docs/noise.md

```diff
@@ -0,0 +1,125 @@
+## Noise
+
+For simulation, it is useful to have `Gate`s that enact noisy quantum evolution.
+Cirq supports modeling noise via *operator sum* representations of
+noise (these evolutions are also known as quantum operations, quantum
+dynamical maps, or superoperators).  This formalism models evolution of the
+density matrix via
+
+![Operator sum representation: $\rho \rightarrow \sum_k A_k \rho A_k^\dagger$](resources/OperatorSumDef.gif)
+
+Where here the A<sub>k</sub> are *Krauss* operators. These operators are not
+necessarily unitary and must satisfy
+the trace preserving property
+
+![Operator sum normalization: $\sum_k A_k^\dagger A_k = I$](resources/OperatorSumNormDef.gif)
+
+As a noisy channel, Krauss operators are not unique. For more details of these
+operators see [John Preskill's notes](http://www.theory.caltech.edu/people/preskill/ph219/chap3_15.pdf).
+
+### Magic methods
+
+A `Gate` can represent a operator sum representation by supporting the
+`channel` protocol.  Alternatively, for channels that represent probabilistic
+mixtures of unitaries, one can implement the `mixture` protocol.
+
+#### cirq.channel and def _channel_
+
+To represent an operator sum evolution, a `Gate` should implement the
+`SupportsChannel` protocol.  To do this, the `Gate` should implement the
+`_channel_(self) -> Sequence[np.ndarray]:` method.  This method should return
+the sequence of `numpy` matrices corresponding to the Krauss operators. The
+basis in which this matrix is expressed is always implicit with respect to
+the object being called. For example, in GateOperations, these matrices
+must be ordered with respect to the list of qubits that the channel is
+applied to. The qubit-to-amplitude order mapping matches the
+ordering of `numpy.kron(A, B)`, where `A` is a qubit earlier in the list
+than the qubit `B`.
+
+If one has defined `_channel_`, then that `Gate` and any `GateOperation`
+that uses that gate can be used as an argument to `cirq.channel` and
+`cirq.channel` will return this sequence of matrices.
+
+Besides objects that support `_channel_`, `cirq.channel` will also fall
+back to other objects that can be interpreted as channels.  For example, if a
+channel is a probabilistic mixture of unitary gates (see below), then
+`cirq.channel` will fall back to seeing the object supports `_mixture_`.
```

Comment:
"seeing the object that supports `_mixture_`" or  "seeing if the object supports `_mixture_`"?
```suggestion
`cirq.channel` will fall back to seeing if the object supports `_mixture_`.
```